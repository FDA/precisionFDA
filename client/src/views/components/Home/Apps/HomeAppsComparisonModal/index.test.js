import React from 'react'
import { shallow } from 'enzyme'

import { HomeAppsComparisonModal } from '.'


describe('HomeAppsComparisonModal test', () => {
  it('should render', () => {
    const component = shallow(<HomeAppsComparisonModal />)

    expect(component).toMatchSnapshot()
  })
})
