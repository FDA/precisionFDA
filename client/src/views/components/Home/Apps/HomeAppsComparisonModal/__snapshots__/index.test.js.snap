// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeAppsComparisonModal test should render 1`] = `
<Modal
  modalFooterContent={<SwitchFooter />}
  title="Attention!"
/>
`;
